packages/api-client/package.json

```diff
@@ -26,8 +26,6 @@
     "@babel/preset-typescript": "7.10.4",
     "@types/jasmine": "3.6.0",
     "@types/karma": "5.0.0",
-    "@types/react": "16.9.19",
```

Comment:
No @types/react anymore?

Reply:
Ya, I'm not 100% sure we can do that but it's worth a try.